browser-test/src/application_download.test.ts

```diff
@@ -168,21 +186,21 @@ describe('normal application flow', () => {
     const demographicsCsvContent = await adminPrograms.getDemographicsCsv()
 
     expect(demographicsCsvContent).toContain(
-      'Opaque ID,Program,Submitter Email (Opaque),TI Organization,Create time,Submit time,csvcurrency (currency),csvdate (date),dropdowncsvdownload (selection),namecsvdownload (first_name),namecsvdownload (middle_name),namecsvdownload (last_name)',
+      'Opaque ID,Program,Submitter Email (Opaque),TI Organization,Create time,Submit time,name (first_name),name (middle_name),name (last_name),csvcurrency (currency),csvdate (date),dropdowncsvdownload (selection)',
```

Comment:
Note to the reviewer: The field ordering issues are here presumably due to the canonical question already being in the database. When publishing, I believe we clone questions that haven't changed. As such, it'd have a later ID and probably would end up being rendered later in the list.